ci_framework/roles/operator_build/tasks/main.yml

```diff
@@ -22,6 +22,11 @@
     - artifacts
     - logs
 
+- name: Initialize role output
```

Comment:
probably need to add `cacheable: true` in order to be passed down from playbook to playbook